GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/task-list.config

```diff
@@ -77,7 +77,7 @@
     {
       "contentTypeKey": "405470f7-9d54-4891-91c9-2efa8dec0833",
       "udi": "umb://element/e10307a5c7ef470996f2b63d3fb97c23",
-      "tasks": "{\r\n  \"layout\": {\r\n    \"Umbraco.BlockList\": [\r\n      {\r\n        \"contentUdi\": \"umb://element/12e95889b5504284981095a61c8b8868\",\r\n        \"settingsUdi\": \"umb://element/bab17a84a7fb4a3f98c7c62ca24ed6b5\"\r\n      },\r\n      {\r\n        \"contentUdi\": \"umb://element/643935631b4146e2b640e4132580a026\",\r\n        \"settingsUdi\": \"umb://element/59ad6b4519614f2bac5f1673e62ab178\"\r\n      },\r\n      {\r\n        \"contentUdi\": \"umb://element/d299b34cca2e47079734a25d20cc9009\",\r\n        \"settingsUdi\": \"umb://element/13b1896f2f094ccb96daeecd8e9033d1\"\r\n      }\r\n    ]\r\n  },\r\n  \"contentData\": [\r\n    {\r\n      \"contentTypeKey\": \"28cfc6cc-1e02-4d9e-8232-b2f921026042\",\r\n      \"udi\": \"umb://element/12e95889b5504284981095a61c8b8868\",\r\n      \"taskName\": \"Thing to do\",\r\n      \"hint\": \"\",\r\n      \"link\": \"[{\\\"name\\\":\\\"#\\\",\\\"url\\\":\\\"#\\\"}]\"\r\n    },\r\n    {\r\n      \"contentTypeKey\": \"28cfc6cc-1e02-4d9e-8232-b2f921026042\",\r\n      \"udi\": \"umb://element/643935631b4146e2b640e4132580a026\",\r\n      \"taskName\": \"Another thing to do\",\r\n      \"link\": \"[{\\\"name\\\":\\\"#\\\",\\\"url\\\":\\\"#\\\"}]\",\r\n      \"hint\": \"\"\r\n    },\r\n    {\r\n      \"contentTypeKey\": \"28cfc6cc-1e02-4d9e-8232-b2f921026042\",\r\n      \"udi\": \"umb://element/d299b34cca2e47079734a25d20cc9009\",\r\n      \"taskName\": \"Yet another thing to do\",\r\n      \"link\": \"[{\\\"name\\\":\\\"#\\\",\\\"url\\\":\\\"#\\\"}]\",\r\n      \"hint\": \"<p>This thing has an <strong>important deadline</strong> you need to know about. It is set to 'Completed' by code in <em>TaskListController.</em></p>\"\r\n    }\r\n  ],\r\n  \"settingsData\": [\r\n    {\r\n      \"contentTypeKey\": \"dac47aa8-2d8c-4024-8b72-118d15916b4c\",\r\n      \"udi\": \"umb://element/bab17a84a7fb4a3f98c7c62ca24ed6b5\",\r\n      \"cssClasses\": \"\",\r\n      \"status\": \"[\\\"Not started\\\"]\"\r\n    },\r\n    {\r\n      \"contentTypeKey\": \"dac47aa8-2d8c-4024-8b72-118d15916b4c\",\r\n      \"udi\": \"umb://element/59ad6b4519614f2bac5f1673e62ab178\",\r\n      \"status\": \"[\\\"Incomplete\\\"]\"\r\n    },\r\n    {\r\n      \"contentTypeKey\": \"dac47aa8-2d8c-4024-8b72-118d15916b4c\",\r\n      \"udi\": \"umb://element/13b1896f2f094ccb96daeecd8e9033d1\",\r\n      \"status\": \"\",\r\n      \"cssClasses\": \"yet-another-thing\"\r\n    }\r\n  ]\r\n}"
```

Comment:
The uSync data for the content type is missing. There is no 'status text' property without it.